dataactvalidator/config/rule_documentation.csv

```diff
@@ -14,8 +14,8 @@ A,A14,GrossOutlayAmountByTAS_CPE= CPE value for GTAS SF 133 line #3020,Calculati
-"A, B",A18,"GrossOutlayAmountByTAS_CPE (file A) = sum of all GrossOutlayAmountByProgramObjectClass (file B) {When not disaggregated by Program Activity and Object Class, this value is the sum of all Gross Outlay Amounts reported in file B, by TAS/Subaccount}",Calculation,Cross File Calculation,Warning,,"Gross Outlay Amount By TAS in File A must match the sum of Gross Outlay Amount By Program Object Class in File B, by TAS.","For the TAS in each submission row in File A, match to same TAS in File B, sum GrossOutlayAmountByProgramObjectClass (including blank prog actv & obj cls), and validate match to GrossOutlayAmountByTAS_CPE in File A.",No,"Applies to CPE version of the field in file B, can disregard ""(including blank prog actv & obj cls)"", sum this across TAS number in file B, total should equal number in file A, TAS numbers should be unique in file A"
-"A, B",A19,ObligationsIncurredTotalByTAS_CPE (File A) = sum of ObligationsIncurredByProgramObjectClass_CPE (File B),Calculation,Cross File Calculation,Warning,,"Obligations Incurred Total By TAS in File A must match the sum of Obligations Incurred By Program Object Class in File B, by TAS.","For the TAS in each submission row in File A, match to same TAS in File B, sum ObligationsIncurredByProgramObjectClass_CPE (including blank prog actv & obj cls), and validate match to ObligationsIncurredTotalByTAS_CPE in File A.",No,Same as A18
+"A, B",A18,"GrossOutlayAmountByTAS_CPE (file A) = sum of all GrossOutlayAmountByProgramObjectClass (file B) {When not disaggregated by Program Activity and Object Class, this value is the sum of all Gross Outlay Amounts reported in file B, by TAS/Subaccount}",Calculation,Cross File Calculation,Warning,,"Gross Outlay Amount By TAS in File A must match the sum of Gross Outlay Amount By Program Object Class in File B, by TAS.","For the TAS in each submission row in File A, match to same TAS in File B, sum GrossOutlayAmountByProgramObjectClass (including blank prog actv & obj cls), and validate match to GrossOutlayAmountByTAS_CPE in File A.",No,Ready
+"A, B",A19,ObligationsIncurredTotalByTAS_CPE (File A) = sum of ObligationsIncurredByProgramObjectClass_CPE (File B),Calculation,Cross File Calculation,Warning,,"Obligations Incurred Total By TAS in File A must match the sum of Obligations Incurred By Program Object Class in File B, by TAS.","For the TAS in each submission row in File A, match to same TAS in File B, sum ObligationsIncurredByProgramObjectClass_CPE (including blank prog actv & obj cls), and validate match to ObligationsIncurredTotalByTAS_CPE in File A.",No,Ready
 A,A20,Must be valid Agency ID in the Common Government-wide Accounting Classification (CGAC),Requirement,External CGAC,Fatal Error,,The Agency ID is defined in CGAC,Match value to CGAC Agency Codes tab in Domain Values file.,Yes,Ready
```

Comment:
Can we change the second to last column ("Implemented") to Yes, since these rules are implemented now?